tests/api/v1/subjects/bulkDelete.js

```diff
@@ -379,6 +388,64 @@ function testBulkDeleteSubjects(jobStatus) {
           [foo.id]: true,
         }));
     });
+    describe('test max subjects per bulk delete >', () => {
+      describe('Max subjects per delete set >', () => {
+        const maxSubjects = 2;
+        beforeEach(() => {
+          sinon
+          .stub(subjectController.helper, 'validateBulkDeleteSubjectSize')
```

Comment:
ur not testing your function and not setting the env variable

Reply:
here the test is a kind of unit test where the helper is the mocked dependency. Maybe here your mock should not care about details, just return true or false and you can test if your controller is behaving as expected (returning proper HTTP status and it's not breaking because something was expected but its not there (undefined things).

I would suggest doing an integrated test where you can change the env var and test HTTP response + unit test to the helper receiving a list of subjects and returning true or false.